assets/xml/objects/object_mm.xml

```diff
@@ -1,68 +1,80 @@
-        <Limb Name="object_mm_Standardlimb_009628" Type="Standard" EnumName="OBJECT_MM_LIMB_05" Offset="0x9628" />
-        <Limb Name="object_mm_Standardlimb_009634" Type="Standard" EnumName="OBJECT_MM_LIMB_06" Offset="0x9634" />
-        <Limb Name="object_mm_Standardlimb_009640" Type="Standard" EnumName="OBJECT_MM_LIMB_07" Offset="0x9640" />
-        <Limb Name="object_mm_Standardlimb_00964C" Type="Standard" EnumName="OBJECT_MM_LIMB_08" Offset="0x964C" />
-        <Limb Name="object_mm_Standardlimb_009658" Type="Standard" EnumName="OBJECT_MM_LIMB_09" Offset="0x9658" />
-        <Limb Name="object_mm_Standardlimb_009664" Type="Standard" EnumName="OBJECT_MM_LIMB_0A" Offset="0x9664" />
-        <Limb Name="object_mm_Standardlimb_009670" Type="Standard" EnumName="OBJECT_MM_LIMB_0B" Offset="0x9670" />
-        <Limb Name="object_mm_Standardlimb_00967C" Type="Standard" EnumName="OBJECT_MM_LIMB_0C" Offset="0x967C" />
-        <Limb Name="object_mm_Standardlimb_009688" Type="Standard" EnumName="OBJECT_MM_LIMB_0D" Offset="0x9688" />
-        <Limb Name="object_mm_Standardlimb_009694" Type="Standard" EnumName="OBJECT_MM_LIMB_0E" Offset="0x9694" />
-        <Limb Name="object_mm_Standardlimb_0096A0" Type="Standard" EnumName="OBJECT_MM_LIMB_0F" Offset="0x96A0" />
-        <Skeleton Name="object_mm_Skel_0096E8" Type="Flex" LimbType="Standard" LimbNone="OBJECT_MM_LIMB_NONE" LimbMax="OBJECT_MM_LIMB_MAX" EnumName="ObjectMmLimb" Offset="0x96E8" />
-        <Animation Name="object_mm_Anim_0099B4" Offset="0x99B4" /> <!-- Original name is "mm_runwait" -->
-        <Animation Name="object_mm_Anim_00A4E0" Offset="0xA4E0" /> <!-- Original name is "mm_tatsu" -->
-        <Animation Name="object_mm_Anim_00A8D8" Offset="0xA8D8" /> <!-- Original name is "mm_uzukumaru" -->
-        <Animation Name="object_mm_Anim_00B09C" Offset="0xB09C" /> <!-- Original name is "mm_wait01" -->
-        <Animation Name="object_mm_Anim_00BA78" Offset="0xBA78" /> <!-- Original name is "mm_watasu" -->
-        <Animation Name="object_mm_Anim_00C32C" Offset="0xC32C" /> <!-- Original name is "mm_watasuwait" -->
-        <Animation Name="object_mm_Anim_00C640" Offset="0xC640" /> <!-- Original name is "mm_yoru" -->
-        <Animation Name="object_mm_Anim_00CD90" Offset="0xCD90" /> <!-- Original name is "uttekure" -->
-        <Animation Name="object_mm_Anim_00DA50" Offset="0xDA50" /> <!-- Original name is "yorokobi_long" -->
+
+        <!-- Postman Animations -->
+        <Animation Name="gPostmanSkippingAnim" Offset="0x468" /> <!-- Original name is "hasiruu" -->
+        <Animation Name="gPostmanHeadInHandsAnim" Offset="0x99C" /> <!-- Original name is "mm_kunou" -->
+        <Animation Name="gPostmanSleepingAnim" Offset="0xFC4" /> <!-- Original name is "mm_neru" -->
+        <Animation Name="gPostmanCheckingMailAnim" Offset="0x1F84" /> <!-- Original name is "mm_postcheck" -->
+        <Animation Name="gPostmanRunningAnim" Offset="0x2238" /> <!-- Original name is "mm_run01" -->
```

Comment:
```suggestion
        <Animation Name="gPostmanSleepAnim" Offset="0xFC4" /> <!-- Original name is "mm_neru" -->
        <Animation Name="gPostmanCheckMailAnim" Offset="0x1F84" /> <!-- Original name is "mm_postcheck" -->
        <Animation Name="gPostmanRunAnim" Offset="0x2238" /> <!-- Original name is "mm_run01" -->
```
We try to avoid `ing` in the names, so the names would look like this. Could you apply that to the other names?